### PR TITLE
Docs changes for GA

### PR DESCRIPTION
**Pull request by theletterf**
Docs changes for GA. See the comments to each file.

### CODE REVIEW

File: MIGRATING.md [hunk 1]
```diff
@@ -70,23 +70,23 @@ Once you have identified an instrumentation package, **install it** using npm or
...
+| SIGNALFX_SERVICE_NAME              | OTEL_SERVICE_NAME                      | |
+| SIGNALFX_ENDPOINT_URL              | _no direct equivalent_                 | see [the notes on endpoint](#endpoint) |
+| SIGNALFX_RECORDED_VALUE_MAX_LENGTH | OTEL_SPAN_ATTRIBUTE_VALUE_LENGTH_LIMIT | |
+| SIGNALFX_TRACING_DEBUG             | _no direct equivalent_                 | see [instrumentation logs](#instrumentation-logs) |
+| SIGNALFX_SPAN_TAGS                 | OTEL_RESOURCE_ATTRIBUTES               | format needs to be changed to `key1=val1,key2=val2` |
+| SIGNALFX_LOGS_INJECTION            | SPLUNK_LOGS_INJECTION                  | |
+| SIGNALFX_LOGS_INJECTION_TAGS       | OTEL_RESOURCE_ATTRIBUTES               | there's no direct equivalent, but values specified in `OTEL_RESOURCE_ATTRIBUTES` will also be used for logs injection |
+| SIGNALFX_ENABLED_PLUGINS           | n/a                                    | see [the README section about instrumentations](./README.md#custom-instrumentation-packages) |
+| SIGNALFX_SERVER_TIMING_CONTEXT     | SPLUNK_TRACE_RESPONSE_HEADER_ENABLED   | |
+| SIGNALFX_TRACING_ENABLED           | OTEL_TRACE_ENABLED                     | |
```
Comment (theletterf) [inline MIGRATING.md:85]: @Rauno56 As discussed, we need the definitive list of settings here.

File: docs/correlate-logs-traces.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# Correlating traces with logs
+
+TODO:
+
+- Attributes that are injected
+- List of supported logging librares
+- Sample formatting and formatting considerations
+- How to enable/disable trace injection into logs
```
Comment (theletterf) [inline docs/correlate-logs-traces.md:8]: @Rauno56 TODO

Comment (Rauno56) [reply docs/correlate-logs-traces.md]: I think we've talked about all of those by now?

**Comment by theletterf** [comment]
Changed default port to 4317.

File: docs/correlate-logs-traces.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
+> The official Splunk documentation for this page is [Connect Node.js trace data with logs](https://docs.splunk.com/Observability/gdi/get-data-in/application/nodejs/instrumentation/connect-traces-logs.html). For instructions on how to contribute to the docs, see [CONTRIBUTING.md](../CONTRIBUTING.md#documentation).
```
Comment (Rauno56) [inline docs/correlate-logs-traces.md:1]: Awesome we've added those comments!

**Review by Rauno56 — APPROVED** [review]

**Pull request merged**